app/modules/site_settings/models.py

```diff
@@ -47,6 +47,49 @@ class SiteSetting(db.Model, Timestamp):
         'email_service_password': {'type': str, 'public': False},
         'email_default_sender_email': {'type': str, 'public': False},
         'email_default_sender_name': {'type': str, 'public': False},
+        'email_header_image_url': {
```

Comment:
Defaults need to be non flukebook specific, but otherwise, yes very much of an improvement.

Reply:
many of these things are set up via the configuration that comes from edm (which pre-dates SiteSettings) and their values (and defaults, even) should come from there.    i am going to take this to slack....